spec/system/admin/products_v3/index_spec.rb

```diff
@@ -377,5 +377,22 @@
         expect(page).to have_field "Name", with: product_by_category.name
       end
     end
+
+    context "with variant tag", feature: :variant_tag do
+      before do
+        create(:variant, tag_list: "organic")
+        create_products 1
+      end
+
+      it "can search by tag" do
+        visit admin_products_url
+        pause
```

Comment:
```suggestion
```

I guess that we want to remove this like `binding.pry`?

Reply:
Yes we should ! I am surprised it's not failing the spec ?